esperanto/malkovro/fajro.md

```diff
@@ -0,0 +1,2 @@
+Vi metas la fajron proksime de la muro , kaj ĝi hardas .
+Ĝi odoris [familiara](http://www.tastespotting.com/features/wp-content/uploads/toasted-marshmallows-for-smores.jpg) , kiel odoro de via infanaĝo ...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.